packages/website/src/components/Navbar/BaseNavbar.tsx

```diff
@@ -0,0 +1,67 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { IconButton } from '@coderscamp/ui/components/IconButton';
+import { Logo } from '@coderscamp/ui/components/Logo';
+import { Menu as ChakraMenu, MenuButton, MenuItem, MenuList } from '@coderscamp/ui/components/Menu';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import { SolidMenuIcon } from '@coderscamp/ui/icons';
+
+import { NavbarItem } from './NavbarItem';
+
+const NavbarElements = [
+  {
+    text: 'Strona główna',
+    destinationLink: '/',
+  },
+  {
+    text: 'Dla mentorów',
+    destinationLink: '/mentorzy',
+  },
+  {
+    text: 'FAQ',
+    destinationLink: '/faq',
+  },
+  {
+    text: 'Kontakt',
+    destinationLink: '/kontakt',
+  },
+];
+
+export const MobileBaseNavbar = () => {
+  const logoLayout = useBreakpointValue({ sm: 'square', md: 'horizontal' } as const);
+  const { route } = useRouter();
+  const fontWeight = (link: string) => (link === route ? 'bold' : 'normal');
+
+  return (
+    <Flex alignItems="center">
+      <ChakraMenu>
+        <MenuButton as={IconButton} aria-label="Options" icon={<SolidMenuIcon />} />
+        <MenuList>
+          {NavbarElements.map((element) => (
+            <Link key={element.text} href={element.destinationLink}>
+              <MenuItem fontWeight={fontWeight(element.destinationLink)}>{element.text}</MenuItem>
+            </Link>
+          ))}
+        </MenuList>
+      </ChakraMenu>
+      {logoLayout && <Logo ml="6" maxWidth="280px" color="black" layout={logoLayout} />}
+    </Flex>
+  );
+};
+
+export const DesktopBaseNavbar = () => {
```

Comment:
🤔 Maybe we should consider add boundary for navbar,  when screen is extra big.

Reply:
Navbar is set on the full width of a page, for high resolution f.e 2500px, it doesn't look good because logo and buttons are near to the edge of the page. The rest of the content is centre and doesn't take full with of page. Maybe we should add a boundary f.e above 2200px resolution navbar should be not on the full width.

![navbar](https://user-images.githubusercontent.com/40128600/128944247-d0bd449a-42fb-4d2c-a89e-d2790cdff619.PNG)

Reply:
Good point, corrected

Reply:
🙃 It'll be great if a navbar has been the same width as the title, but it's just a cosmetic. I give approval for it.